goon/browserassets/css/browserOutput.css

```diff
@@ -32,6 +32,42 @@ a {color: #0000ff;}
 a:visited {color: #ff00ff;}
 a.popt {text-decoration: none;}
 
+button {
+    display: flex;
+    align-items: center;
+    overflow: hidden;
+
+    /* margin: 10px; */
+    padding: 6px 6px;
+
+    cursor: pointer;
+    user-select: none;
+    transition: all 150ms linear;
+    text-align: center;
+    white-space: nowrap;
+    text-decoration: none !important;
+    text-transform: none;
+    text-transform: capitalize;
+
+    color: #202129;
+    background-color: #ffffff;
+    border: 0 none;
+    border-radius: 4px;
+
+    font-size: 13px;
+    font-weight: 500;
+    line-height: 1.3;
+
+    opacity: 0.5;
+}
+
+button:hover {
+    transition: all 150ms linear;
+    color: #202129;
+    background-color: #e1e2e2;
+    opacity: 1;
+}
+
```

Comment:
This all needs adding to `browserOutput-dark.css`

Reply:
We can probably do something pretty dynamic with dark mode. Will definitely research the dark mode options if this is approved